File: app/src/at-hungtran/java/asiantech/internship/summer/recyclerview/TimelineActivity.java
package asiantech.internship.summer.recyclerview;

import android.support.v4.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import asiantech.internship.summer.R;

public class TimelineActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_timeline);
        initFragment();
    }
    private void initFragment() {
        FragmentTransaction fragmentTransaction = getSupportFragmentManager().beginTransaction();
        fragmentTransaction.add(R.id.fragmentContainer, new TimelineFragment());
        fragmentTransaction.commit();
    }
}

File: app/src/at-hungtran/java/asiantech/internship/summer/kotlin/recyclerview/TimelineActivity.kt
package asiantech.internship.summer.kotlin.recyclerview

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import asiantech.internship.summer.R
import asiantech.internship.summer.recyclerview.TimelineFragment

class TimelineActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_timeline)
        initFragment()
    }

    private fun initFragment() {
        val fragmentTransaction = supportFragmentManager.beginTransaction()
        fragmentTransaction.add(R.id.fragmentContainer, TimelineFragment())
        fragmentTransaction.commit()
    }
}
